tests/www/presenters/test_get_statistics_presenter.py

```diff
@@ -20,6 +20,7 @@
     planned_resources=Decimal("400.1"),
     planned_means=Decimal("215.23"),
     payout_factor=Decimal("0.74516"),
+    fpc_balance=Decimal("8.76"),
```

Comment:
In this module are missing tests for the correct display of this fpc_balance number.

Reply:
done